galasa-extensions-parent/dev.galasa.extensions.common/bnd.bnd

```diff
@@ -0,0 +1,15 @@
+-snapshot ${tstamp}
+Bundle-Name: dev.galasa.extensions.common
+Bundle-Description: Provides access to the CPS via a remote REST endpoint over http. 
+Bundle-License: https://www.eclipse.org/legal/epl-2.0
+Export-Package: dev.galasa.extensions.common.*
+Import-Package: \
+dev.galasa,\
+dev.galasa.framework.spi,\
+javax.net.ssl,\
+javax.security.cert,\
+org.apache.commons.logging,\
+org.apache.http.impl.client
+Embed-Transitive: true
```

Comment:
Embedding these dependencies many times is bad. Separate item added to the story to sort this out.